srcalc/src/main/java/gov/va/med/srcalc/domain/model/MultiSelectVariable.java

```diff
@@ -55,7 +49,7 @@ public MultiSelectVariable(
     {
         super(displayName, group, key);
         setDisplayType(displayType);
-        setOptions(options);
+        fOptions = new ArrayList<>(options);
```

Comment:
I don't think so. There's no technical reason to prevent that. I will not allow that on the Admin UI, though.